setup.py

```diff
@@ -1,37 +1,36 @@
 import pathlib
```

Comment:
The name of this file is a bit ambiguous, setup what? Let's add comments at the top explaining what we want this file to do, or rename it to be more specific and avoid confusion.

Reply:
Resolved in PR #51